features/org.mozilla.xulrunner.feature/pom.xml

```diff
@@ -26,6 +26,11 @@
 						</goals>
 					</execution>
 				</executions>
+				<configuration>
+					<excludes>
+						<plugin id="org.mozilla.xulrunner.cocoa.macosx"/>
```

Comment:
cause we do have only x86 builded xulrunner for OSX. So we don't need OSX xulrunner bundles.